packages/sanity/package.json

```
@@ -158,6 +158,7 @@
     "@sanity/icons": "^3.0.0",
     "@sanity/image-url": "^1.0.2",
     "@sanity/import": "^3.37.3",
+    "@sanity/insert-menu": "^1.0.1",
```

```suggestion
    "@sanity/insert-menu": "1.0.1",
```

After discussing with team suggest we pin this version in accordance with `@sanity/presentation` and likely other externalized dependencies going forward

---

Thanks! I've pinned it now.